docs/accessControl.md

```diff
@@ -202,7 +202,7 @@ To block students from viewing closed assessment scores, set `"showClosedAssessm
 
 Note that the `showClosedAssessment` and `showClosedAssessmentScore` statements are listed twice above, once while the assessment is in progress (`startDate` to `endDate`) and once in a second rule that will take effect after the `endDate`. The additional `"active": false` restriction prevents students from starting the exam after the `endDate`.
 
-The `showClosedAssessmentScore` access rule restriction is often used in conjunction with [disabling real-time grading](assessment.md#disabling-real-time-grading) and setting `"showClosedAssessment": false`. It may also be used in assessments with [manual grading](manualGrading.md), to hide the assessment grade until all students have been graded.
+The `showClosedAssessmentScore` access rule restriction is often used in conjunction with [disabling real-time grading](assessment.md#disabling-real-time-grading) and setting `"showClosedAssessment": false`. It may also be used in assessments with [manual grading](manualGrading/), to hide the assessment grade until all students have been graded.
```

Comment:
I'm not sure if this is an issue just with preview builds, but the page that this links to in the preview build (https://prairielearn--9484.org.readthedocs.build/en/9484/accessControl/manualGrading/) does not exist. The presence of `/accessControl/` in the URL is the problem

I don't know if this would fix this or not, but regardless, consider using `manualGrading/index.md` so that Cmd+click in an editor goes straight to the file.

Reply:
Fixed by 5bf438a.